shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -283,6 +283,7 @@ object GlutenConfig {
 
   var GLUTEN_ENABLE_BY_DEFAULT = true
   val GLUTEN_ENABLE_KEY = "spark.gluten.enabled"
+  val GLUTEN_ENABLE_FOR_THREAD_KEY = "spark.gluten.enabledForThisThread"
```

Comment:
Please add some comments here to help reader figure out its usage. It would also be nice if its usage can be documented in gluten doc. Maybe, `Configuration.md` is a good place.

Reply:
Already update `Configuration.md`